python3/libexec/nbd_client_manager.py

```diff
@@ -191,6 +195,8 @@ def connect_nbd(path, exportname):
     """Connects to a free NBD device using nbd-client and returns its path"""
     # We should not ask for too many nbds, as we might not have enough memory
     _call(["modprobe", "nbd", "nbds_max=24"])
+    # Wait for systemd-udevd to process the udev rules
+    _call(["udevadm", "settle", "--timeout=30"])
```

Comment:
From the document, would `udevadm wait --settle --timeout=30 [device|syspath] ...` be more specific for the purpose here?

Reply:
Thanks for the inspiration.  Considering what `copilot` said, I will just keep current solution.:
```
udevadm wait --settle:

This command is not standard and may not be available in all versions of udevadm. It is likely a typo or confusion with the udevadm settle command.
If you encounter this command in documentation or scripts, it is advisable to replace it with udevadm settle```